src/main/java/com/aws/greengrass/shadowmanager/ShadowManager.java

```diff
@@ -239,166 +239,178 @@ protected void install(InstallConfig installConfig) {
         }
 
         inboundRateLimiter.clear();
+        config.lookupTopics(CONFIGURATION_CONFIG_KEY).subscribe((what, newv) -> {
+            if (what.equals(WhatHappened.timestampUpdated)) {
+                return;
+            }
+            if (installConfig.configureSynchronizeConfig) {
+                configureSynchronization(newv);
```

Comment:
`newv` will be null on the first call. Can we avoid passing null? Just call these configure methods with no parameters and let them do the right thing by reading the current config state.

Reply:
Since we're subscribing to the top level configuration key, running all of these configure methods even when their keys don't change is not needed. 

I'm handling null in the configure methods but I do get your comment on not passing null.